integration/manifests/netwatcher/0netwatcher_rbac.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: netwatcher
+  namespace: kube-system
+  labels:
+      kubernetes.io/cluster-service: "true"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    kubernetes.io/bootstrapping: rbac-defaults
+  name: system:netwatcher
+rules:
+- apiGroups:
+  - "danm.k8s.io"
+  resources:
+  - danmnets
```

Comment:
I think Pod will be also needed. CNI reads the Pod for its annotation

Reply:
Buuuuut.... this one is only for netwatcher.
CNI is configured through the /etc/cni/net.d config for kubelet, or not? For containerized kubelet, it also uses the local /etc/kubernetes/kubelet.conf and not ServiceAccounts, I would think.

Reply:
yes, I'm just being stupid again